### PR TITLE
Add example to :class:`FunctionGraph`

### PR DESCRIPTION
**Pull request by jojo96**
## Changelog / Overview

Added example for FunctionGraph class as there were no previous examples

## Motivation

Adding more examples so that the library is easier for the new users
## Explanation for Changes

Added more examples for function graph class

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Review by Nikhil0504 — APPROVED** [review]
LGTM but it would be presentable if you could make the 3 parametric function to have separate `self.add()` functions.

**Comment by jojo96** [comment]
> LGTM but it would be presentable if you could make the 3 parametric function to have separate `self.add()` functions.

Added the separate `self.add()` functions.

**Review by hydrobeam — CHANGES_REQUESTED** [review]
Some comments about formatting. `FunctionGraph` does seem exactly identical to `ParametricFunction` in terms of code 😅. Although, this example does look cool

File: manim/mobject/functions.py [hunk 1]
```diff
@@ -103,6 +103,23 @@ def generate_points(self):
...
+        :save_last_frame:
+
+        class ExampleFunctionGraph(Scene):
+            def construct(self):
+                cos_func = lambda t: np.cos(t) + 0.5 * np.cos(7 * t) + (1 / 7) * np.cos(14 * t)
+                sin_func_1 = lambda t: np.sin(t) + 0.5 * np.sin(7 * t) + (1 / 7) * np.sin(14 * t)
+                sin_func_2 = lambda t: np.sin(t) + 0.5 * np.sin(7 * t) + (1 / 7) * np.sin(14 * t)
+                self.add(FunctionGraph(cos_func, color=RED))
+                self.add(FunctionGraph(sin_func_1, color=BLUE))
+                self.add(FunctionGraph(sin_func_2, x_range=np.array([-8, 8, 0.1]), color=GREEN).move_to([0, 1, 0]))
```
Comment (hydrobeam) [inline manim/mobject/functions.py:120]: Applies black formatting and suggestions from code review. If you disagree with the suggestion please let me know 👍
```suggestion
        class ExampleFunctionGraph(Scene):
            def construct(self):
                cos_func = FunctionGraph(
                    lambda t: np.cos(t) + 0.5 * np.cos(7 * t) + (1 / 7) * np.cos(14 * t),
                    color=RED,
                )

                sin_func_1 = FunctionGraph(
                    lambda t: np.sin(t) + 0.5 * np.sin(7 * t) + (1 / 7) * np.sin(14 * t),
                    color=BLUE,
                )

                sin_func_2 = FunctionGraph(
                    lambda t: np.sin(t) + 0.5 * np.sin(7 * t) + (1 / 7) * np.sin(14 * t),
                    x_range=[-4, 4],
                    color=GREEN,
                ).move_to([0, 1, 0])

                self.add(cos_func, sin_func_1, sin_func_2)
```

**Review by hydrobeam — COMMENTED** [review]
Black formatting:

[link to example](https://manimce--1637.org.readthedocs.build/en/1637/reference/manim.mobject.functions.FunctionGraph.html?highlight=functiongraph#manim.mobject.functions.FunctionGraph)

**Comment by jojo96** [comment]
> @jojo96 : Please have a look at the suggested changes, and click on "commit suggestion", if you agree on the changes

Hi, thanks for the suggestion. Just committed it.

**Comment by hydrobeam** [comment]
LGTM. Thank you for the contribution!

**Review by hydrobeam — APPROVED** [review]

**Pull request merged**